Umbraco-Cloud/Troubleshooting/Deployments/Structure-Error/index.md

```diff
@@ -24,94 +20,124 @@ Example:
 
-![Before extraction error](images/visualization1.png)
+## Deciding which file you want to use
 
-Up until now, this project has been working fine, since no deployments have been made from Development to Live since the Document Type was created on Development.
+In order to fix this problem, you will have to decide which of the colliding entities is the correct one and the one you want to use on your Live environment.
 
-### Deploying your changes
+Let's use the example from the beginning of this article, where two `.uda` files for the Document Type "home" are colliding.
 
-It’s now time to deploy the newest changes to the Live environment. Since a Document Type with the same alias has been created in both the Development and the Live environment, the deployment will fail.
+    Some artifacts collide on unique identifiers.
+    This means that they have different Udis, yet
+    they refer to the same unique Umbraco object
+    and therefore cannot be processed.
+    ---------------------------------------------
+    Collisions for entity type "document-type": 
+      Collisions for unique identifier "home":
+        UdaFile: ~/data/revision/document-type__4c04d968448747d791b5eae254afc7ec.uda
+        UdaFile: ~/data/revision/document-type__f848c577f02b4ee5aea84f87458072a4.uda
 
-![After extraction error](images/visualization2.png)
+For this example, it’s decided that the Document Type currently used on the Live environment is the one we want to use going forward.
 
-On a deployment between Umbraco Cloud environments, all the `.uda` files in the `/data/revision` folder will get synced. For this project, this means that both the Development and the Live environments will have two different `.uda` files for the Document Type – the only thing that’s different between the two files are the GUID since they were created in different environments.
+![Visual representation of the error message](images/visualizing.png) - *A visual representation of the error message*
 
-**NOTE**: This is when you will see an extraction error like the one shown at the beginning of this article. It is simply not possible to add types with the same alias in the database.
+In order to figure out which of the two colliding `.uda` files is the one for the Document Type being used on the Live environment follow these steps:
 
-### Choosing the correct UDA file
+1. Access **Kudu** on the Live environment
+2. Use the CMD console (found under the 'Debug console' menu) to navigate to your `site/wwwroot/data/revision` folder
+3. Remove the colliding `.uda` files mentioned in the error message
+4. Go back to the `/wwwroot/data` folder and run this command: `echo > deploy-export` in the console
+5. This will regenerate a `.uda` file for the Document Type being used on the Live environment
```

Comment:
```suggestion
5. This will regenerate all `.uda` files for the Live environment - this means only the currently used one will be there afterwards
```